src/backend/libs/efiling-account-client/src/main/java/ca/bc/gov/open/jag/efilingaccountclient/config/AutoConfiguration.java

```diff
@@ -41,6 +41,7 @@ public AccountFacadeBean accountFacadeBean() {
     public RoleRegistryPortType roleRegistryPortType() {
         JaxWsProxyFactoryBean jaxWsProxyFactoryBean = new JaxWsProxyFactoryBean();
         jaxWsProxyFactoryBean.setServiceClass(RoleRegistryPortType.class);
```

Comment:
the class could be part of the property object so you could instantiate a `JaxWsProxyFactoryBean` by client name

Reply:
and remove some code

Reply:
I think it would only remove the instantiation. Maybe we could move the creation to the properties. Pass it the class to build.